lib/compress/huf_compress.c

```diff
@@ -169,7 +169,7 @@ size_t HUF_writeCTable (void* dst, size_t maxDstSize,
 }
 
 
-size_t HUF_readCTable (HUF_CElt* CTable, unsigned* maxSymbolValuePtr, const void* src, size_t srcSize)
+size_t HUF_readCTable (HUF_CElt* CTable, unsigned* maxSymbolValuePtr, const void* src, size_t srcSize, size_t* hasNoZeroWeights)
```

Comment:
Nit: `hasNoZeroWeights` should be an `int` or an `unsigned`. It is unusual to use a `size_t` as a boolean type.

Reply:
I would also recommend making the parameter `hasZeroWeights`, to avoid a negative. Generally, it is best to keep boolean variables as positive (`X` instead of `notX`).